Orange/projection/pca.py

```diff
@@ -152,6 +155,38 @@ def partial_fit(self, data):
         return self
 
 
+class TruncatedSVD(SklProjector, _FeatureScorerMixin):
+    __wraps__ = skl_decomposition.TruncatedSVD
+    name = 'Truncated SVD'
+    supports_sparse = True
+
+    def __init__(self, n_components=None, algorithm='randomized', n_iter=5,
+                 random_state=None, tol=0.0, preprocessors=None, max_components=None):
+        super().__init__(preprocessors=preprocessors)
+        if n_components is not None and max_components is not None:
+            raise ValueError("n_components and max_components can not both be defined.")
+        # max_components limits the number of SVD components if the minimum
+        # shape of the X matrix (after preprocessing) is higher than
+        # max_components, so that sklearn does not always compute the full
+        # transform, which is faster and uses less memory for big data.
+        self.max_components = max_components
```

Comment:
I'd have `max_components` removed (here and in `PCA` above). No good reason for it. None.